frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -55,6 +58,23 @@ function EventCard({
   || data.status === TRAINING_REPORT_STATUSES.SUSPENDED);
   const menuItems = [];
 
+  const canCompleteEvent = (() => {
+    if (!isOwner && !isNotCompleteOrSuspended) {
+      return false;
+    }
+
+    if (!trainingReportComplete) {
```

Comment:
I don't think we should have the ! not on this IF. We only want to add the 'Complete' option if it't not currently completed.

Reply:
It's not the status, it's whether the event has been submitted. I can change the name if you find it confusing.

Reply:
Should I be able to complete a 'Completed' event?

![image](https://github.com/user-attachments/assets/30e8fbe8-4e43-4651-88da-4cbb8ca81ff1)